src/control/lib/spdk/ctests/nvme_discover_test.c

```diff
@@ -0,0 +1,175 @@
+/**
```

Comment:
Any reason not to build/run these tests for CI?

Reply:
just haven't finished the PR yet